docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -161,6 +162,27 @@ image::images/alert-change-status.png[Bulk action menu with multiple alerts sele
 
 * In an alert's details flyout, click *Take action* and select a status.
 
+[float]
+[[apply-alert-tags]]
+==== Apply and filter alert tags
+
+Use alert tags to organize related alerts into categories that you can filter and group. For example, use the `False Positive` alert tag to label a group of alerts as false positives. Then, search for them by entering the `kibana.alert.workflow_tags : "False Positive"` query into the KQL bar.  
```

Comment:
@joepeeples good point. I forgot that the Alert table has additional filters below the KQL bar. It could definitely be worth mentioning as an example of how to filter for tagged alerts.  

@dplumlee, couple questions for you: 
I added a custom control that filters alerts by the `kibana.alert.workflow_tags` field and here's what the list of options looks like:

<img width="900" alt="Screenshot 2023-07-19 at 4 47 00 PM" src="https://github.com/elastic/security-docs/assets/79124755/bd45309a-99de-43a8-87e8-a0762d9416be">

Does the `Exists` option display all alerts with tags (i.e., alerts where tags exist)? Also, is the number of options correct? Should it say that users can select from 3 options, instead of 2?

Reply:
@dplumlee can confirm, but that's how I understand the `Exists` option to work. And my guess is that the `2 options` tally in the drop-down refers to the number of field values that you can filter by, not the total number of things you can select in the drop-down. (I think some fields might allow additional things like "Does not exist" which would also not be counted in the tally?)

Reply:
Yeah the `Exists` option just displays alerts that have tags on them. And Joe is right about the number of options, it's the number of unique values that exist in all the tags arrays. If you had just `False Positive` it would say 1, `Exists` isn't a value it's just a shortcut for that `: *` filter

Reply:
Thanks both for the clarifications! @joepeeples I'll add in a line about filtering for tagged alerts via the controls above the Alerts table.

Reply:
```suggestion
Use alert tags to organize related alerts into categories that you can filter and group. For example, use the `False Positive` alert tag to label a group of alerts as false positives. Then, search for them by entering the `kibana.alert.workflow_tags : "False Positive"` query into the KQL bar. Alternatively, use the Alert table's <<drop-down-filter-controls,drop-down filters>> to filter for tagged alerts. 
```